shared/XrUtility/XrSceneUnderstanding.h

```diff
@@ -18,7 +18,8 @@
 #include "XrStruct.h"
 #include "XrHandle.h"
 #include "XrMath.h"
-#include "XrExtensionContext.h"
+#include "XrExtensions.h"
+#include <ostream>
```

Comment:
This header uses [`std::basic_ostream`](https://en.cppreference.com/w/cpp/io/basic_ostream) in the last function. Our sample was probably including `<ostream>` or `<iostream>` somewhere else but if someone copies this header to their own project they might get a compiler error if they weren't already including `<ostream>`.

Alternatively we could move the function (ReadSerializedScene) to Scene_PlaneFinding instead of putting it in this header.